test/header_parser_tests/separate_definition_test.dart

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2020, the Dart project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:ffigen/src/code_generator.dart';
+import 'package:ffigen/src/config_provider.dart';
+import 'package:ffigen/src/header_parser.dart' as parser;
+import 'package:ffigen/src/strings.dart' as strings;
+import 'package:logging/logging.dart';
+import 'package:test/test.dart';
+import 'package:yaml/yaml.dart' as yaml;
+
+import '../test_utils.dart';
+
+late Library actual, expected;
+
+void main() {
+  group('separate_definition', () {
+    setUpAll(() {
+      logWarnings(Level.SEVERE);
+    });
+    test('different header order', () {
+      final entryPoints = [
+        "test/header_parser_tests/separate_definition_base.h",
```

Comment:
Can we make two tests, also one with the order of the entryPoints reversed? Then we test that it really doesn't matter what the order is.

Reply:
The current test will generate 2 bindings, one with headers - h1, h2 and one with headers - h2, h1

This should ensure that different header order will generate the same bindings.

What other test should I add?

Reply:
Oh, I was still very sleepy this morning. You're already doing `entryPoints.reversed`! 😄

Reply:
😆